docs/user/helper.rst

```diff
@@ -19,6 +19,35 @@ length of the arrays and not the number of keys, and you get a
 with :class:`.SliceDict`, this works.
 
 
+.. _slicedictx:
+
+SliceDatasetX
+-------------
+
+A :class:`.SliceDatasetX` is a wrapper for
+:class:`torch.utils.data.Dataset`\s that makes them behave a little
```

Comment:
I think in order to avoid convoluted names in the running text, our convention was to use unqualified name and spell PyTorch in front, i.e. in this case:

> is a wrapper for PyTorch :class:`~torch.utils.data.Dataset`s

Reply:
Will fix it.